docs/guides/cortex-with-cassandra.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Running Cortex with Cassandra"
+linkTitle: "Running Cortex with Cassandra"
+weight: 7
+slug: cassandra
+---
+
+This guide covers how to run a single local Cortex instance - with the chunks storage engine - storing time series chunks and index in Cassandra.
+
+In this guide we're going to:
+
+1. Setup a locally running Cassandra
+2. Configure Cortex to store chunks and index on Cassandra
+3. Configure Prometheus to send series to Cortex
+4. Configure Grafana to visualise metrics
+
+## Setup a locally running Cassandra
+
+Run Cassandra with following cmd:
+
+```
+docker run -d --name cassandra --rm -p 9042:9042 cassandra:3.11
+```
+
+Get a cql shell in the Cassandra container:
+
+```
+docker exec -it <container_id> cqlsh
+```
+
+Create a new Cassandra keyspace for Cortex metrics:
```

Comment:
These commands are likely to be run by people with no previous Cassandra experience, it would be good to shortly (at most one or two sentences) explain what is a keyspace and why it's needed.